lib/puppet/type/java_ks.rb

```diff
@@ -152,6 +152,13 @@ def self.title_patterns
         ]
       ],
       [
+        /^(.*):([a-z]:(\/|\\).*)$/i,
```

Comment:
No we need to check for second ':' otherwise where it splits is the second ':' not the first leaving the trailing ':{drive_letter}' to the alias and not the destination